controller/controller.Employee.js

```diff
@@ -0,0 +1,49 @@
+  /* eslint-disable no-plusplus */
+  /* eslint-disable no-await-in-loop */
+
+  const signupEmployeeService = require('../services/services.signupEmployee')();
+
+  module.exports = () => {
+
+     /*
+      *Add Employee
+      */
+
+     const signupEmployee = async (req, res, next, {
+        logger,
+        db
+     }) => {
+        try {
+           const payload = req.body;
+
+           const response = await signupEmployeeService.signupEmployee(payload, logger, db);
+           if (response == 'foundEmail') {
+              res.status(200).send({
+                 status: '200',
+                 result: 'Email Id Already Exists'
```

Comment:
better response message. 

EX: ${payload.email} already exists in the database.